v1/metrics.yaml

```diff
@@ -429,115 +439,860 @@ paths:
+          type: string
+
+  pageview-article:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access:
+              type : string
+            article:
+              type: string
+            agent:
+              type: string
+            granularity:
+              type: string
+            timestamp:
+              # the hourly timestamp will be stored as YYYYMMDDHH
+              type: string
+            views:
+              type: integer
+              format: int64
+
+  pageview-project:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access:
+              type : string
+            agent:
+              type: string
+            granularity:
+              type: string
+            timestamp:
+              # the hourly timestamp will be stored as YYYYMMDDHH
+              type: string
+            views:
+              type: integer
+              format: int64
+
+  pageview-tops:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type: string
+            access:
+              type : string
+            year:
+              type: string
+            month:
+              type: string
+            day:
+              type: string
+            articles:
+              # format for this is a json array: [{rank: 1, article: <<title>>, views: 123}, ...]
+              type: string
+
+  unique-devices:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access-site:
+              type : string
+            granularity:
+              type: string
+            timestamp:
+              # the daily timestamp will be stored as YYYYMMDD
+              type: string
+            devices:
+              type: integer
+              format: int64
+
+  pagecounts-project:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            access-site:
+              type : string
+            granularity:
+              type: string
+            timestamp:
+              type: string
+            count:
+              type: integer
+              format: int64
+
+  new-pages:
+    properties:
+      items:
+        type: array
+        items:
+          properties:
+            project:
+              type : string
+            editor-type:
+              type: string
+            page-type:
+              type: string
+            granularity:
+              type: string
+            results:
+              type: array
+              items:
+                properties:
+                  timestamp:
+                    type: string
+                  new_pages:
```

Comment:
Why `pageS` is it's a single int?

Reply:
Because it's actually a number of pages.

Reply:
Oh, gotcha.. I thought it's like a list with page_ids